### PR TITLE
Master+new download page

### PR DESCRIPTION
**Pull request by ddurieux**

### CODE REVIEW

File: downloads.md [hunk 1]
```diff
@@ -8,29 +8,66 @@ order: 4
...
+* Trusty 
+* xenial
+* Yakkety
+* Zesty
+* Artful
+
+
+### RPM
+
+There is no possibility for the moment with development PRM packages
```
Comment (mohierf) [inline downloads.md]: It is not possible currently with RPM packages.

File: downloads.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ Add in your `/etc/apt/sources.list`:
 ```
 deb https://dl.bintray.com/ddurieux/alignak_deb-stable {distribution} main
 ```
-where `{distribution}` is your distribution name, wokr only for one of the list:
+where `{distribution}` is your distribution name, work only for one in the list:
```
Comment (mohierf) [inline downloads.md]: , only for a distribution in this list:

File: downloads.md [hunk 3]
```diff
@@ -62,7 +62,7 @@ where `{distribution}` is your distribution name, wokr only for one of the list:
 
 ### RPM
 
-There is no possibility for the moment with development PRM packages
+There is no possibility for the moment to have RPM packages of development version
```
Comment (mohierf) [inline downloads.md]: It is not possible currently to have the develop branch version with RPM packages.

**Comment by ddurieux** [comment]
Changes done ;)

**Pull request merged**